db/seeds.rb

```diff
@@ -1,3 +1,50 @@
+require 'csv'
+
+csv_market = File. read ('./seed_csvs/markets.csv')
+csv_market = CSV.parse(csv_market)
+csv_market.each do |row|
+  m= Market.new
```

Comment:
When seeding out database from the CSV files, we should use the `id` column from the CSV file, so we can be sure our database is using the correct IDs.
We should add a line like `m.id = row[0]` after this line, and similar lines for Vendor, Product, and Sale.